src/groups/mqb/mqbcfg/mqbcfg_messages.cpp

```diff
@@ -6330,12 +5804,12 @@ Configuration::print(bsl::ostream& stream, int level, int spacesPerLevel) const
 }  // close package namespace
 }  // close enterprise namespace
 
-// GENERATED BY BLP_BAS_CODEGEN_2023.10.07
+// GENERATED BY @BLP_BAS_CODEGEN_VERSION@
```

Comment:
This looks suspicious.

Reply:
I am using the newest version of BAS codegen. For some reason, this string substitution doesn't work but shouldn't be a big deal.